app/UserPreferences.m

```diff
@@ -184,9 +211,20 @@ - (instancetype)init {
             kPreferenceDisableDimmingKey: property(shouldDisableDimming),
             kPreferenceLaunchCommandKey: property(launchCommand),
             kPreferenceBootCommandKey: property(bootCommand),
+            kPreferenceCursorStyleKey: property(cursorStyle),
+            kPreferenceBlinkCursorKey: property(blinkCursor),
             kPreferenceHideStatusBarKey: property(hideStatusBar),
+            kPreferenceColorSchemeKey: property(colorScheme),
+            // This one is a little bit special, so it needs extra handling.
+            // The backing property for this is intentionally underscored.
+            kPreferenceThemeKey: @"userTheme",
         };
 #undef property
+        
+        [self updateTheme];
+        
+        [NSNotificationCenter.defaultCenter addObserver:self selector:@selector(updateTheme:) name:ThemesUpdatedNotification object:nil];
+        [NSNotificationCenter.defaultCenter addObserver:self selector:@selector(updateTheme:) name:ThemeUpdatedNotification object:nil];
```

Comment:
What's the difference between these two notifications?

Reply:
The former is when the list of themes changes. The latter is when the current theme changes.